modules/elasticsearch/src/main/java/org/testcontainers/elasticsearch/ElasticsearchContainer.java

```diff
@@ -29,10 +29,14 @@
     private static final int ELASTICSEARCH_DEFAULT_TCP_PORT = 9300;
 
     /**
-     * Elasticsearch Docker base image
+     * Elasticsearch Docker base image: default distribution with built-in free basic elastic license
      */
-    private static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse("docker.elastic.co/elasticsearch/elasticsearch");
-    private static final DockerImageName DEFAULT_OSS_IMAGE_NAME = DockerImageName.parse("docker.elastic.co/elasticsearch/elasticsearch-oss");
+    public static final DockerImageName ELASTICSEARCH_IMAGE = DockerImageName.parse("docker.elastic.co/elasticsearch/elasticsearch");
+
+    /**
+     * Elasticsearch Docker base image: oss distribution under Apache2 license
+     */
+    public static final DockerImageName ELASTICSEARCH_OSS_IMAGE = DockerImageName.parse("docker.elastic.co/elasticsearch/elasticsearch-oss");
```

Comment:
This image has been removed after 7.10.2. So I believe that I should update the PR and remove entirely this field.

Reply:
Aren't images distributed over Docker Hub again?

Reply:
Nope. There's no -oss package after 7.10.2

Reply:
But there's the default distribution as it has always been 😊